package.json

```diff
@@ -207,18 +207,34 @@
           "type": "string",
           "enum": [
             "ignore",
-            "forceQuarkus",
```

Comment:
what happens if you already had `forceQuarkus` set?

Reply:
well the good news is unknown values are considered `force`, since the code doesn't check it, so, no regression there; the bad news is unknown values are considered `force`, since the code doesn't check it, so, not great.

Reply:
Could you be a bit more specific? Are you saying that now it's attempting to force non-qute supported languages as well?